### PR TITLE
increase qps and burst

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

**Comment by skeeey** [comment]
refer to https://github.com/open-cluster-management/backlog/issues/7072

File: pkg/hub/manager.go [hunk 1]
```diff
@@ -18,21 +18,28 @@ import (
 	"github.com/open-cluster-management/registration/pkg/hub/rbacfinalizerdeletion"
 
 	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/rest"
 )
 
 // RunControllerManager starts the controllers on hub to manage spoke cluster registration.
 func RunControllerManager(ctx context.Context, controllerContext *controllercmd.ControllerContext) error {
-	kubeClient, err := kubernetes.NewForConfig(controllerContext.KubeConfig)
+	// Increase the QPS and Burst to enhance the ability of kube client to hanle requstes requests in concurrent
```
Comment (qiujian16) [inline pkg/hub/manager.go]: typo handle requests

File: pkg/hub/manager.go [hunk 2]
```diff
@@ -18,21 +18,28 @@ import (
...
+	// TODO: these may be tuned by the flag of library-go when new a controller command in the future
```
Comment (qiujian16) [inline pkg/hub/manager.go]: change some wording here. Something like:

Use flag to change qps/burst

File: pkg/hub/manager.go [hunk 3]
```diff
@@ -18,21 +18,28 @@ import (
...
+	kubeConfig := rest.CopyConfig(controllerContext.KubeConfig)
+	kubeConfig.QPS = 100.0
```
Comment (qiujian16) [inline pkg/hub/manager.go]: maybe we should  set this only when QPS in kubconfig is not set.

File: pkg/hub/manager.go (hunk 1 above)
Comment (skeeey) [reply pkg/hub/manager.go]: corrected

File: pkg/hub/manager.go (hunk 2 above)
Comment (skeeey) [reply pkg/hub/manager.go]: changed

File: pkg/hub/manager.go (hunk 3 above)
Comment (skeeey) [reply pkg/hub/manager.go]: add a judgement for it

**Pull request merged**